xonsh/built_ins.py

```diff
@@ -466,6 +467,8 @@ def run_subproc(cmds, captured=False):
             stderr = builtins.__xonsh_stderr_uncaptured__
         uninew = (ix == last_cmd) and (not _capture_streams)
         alias = builtins.aliases.get(cmd[0], None)
+        if alias is None and cmd[0] in builtins.__xonsh_enabled_builtin_commands__:
+            alias = all_builtin_commands[cmd[0]]
```

Comment:
I think it's more conventional to draw a distinction between the built-in commands and user-defined aliases.  I feel like this distinction is made in most other shells, at least...

Then again, we already kind of abuse that distinction with aliases that are implemented as Python functions.

I would personally like to move a lot of things from `aliases` to built-in commands, as that makes more sense to me.

This structure also makes it easier to implement the `enable` command.